prov/rxd/src/rxd.h

```diff
@@ -364,6 +364,12 @@ static inline void *rxd_pkt_start(struct rxd_pkt_entry *pkt_entry)
 	return (void *) ((char *) pkt_entry + sizeof(*pkt_entry));
 }
 
+static inline void rxd_dequeued_pkt_entry(struct rxd_pkt_entry *pkt_entry)
```

Comment:
I don't love the name of the new function. Can we rename it to something like
rxd_remove_free_pkt_entry ?
I want to make it clear that it is also freeing the buffer.

Reply:
@aingerson Thanks. Done.
Renamed the function to `rxd_remove_free_pkt_entry`.